TextInputKit/TextInputKit/Code/SpecializedTextInput/CardNumber/BankCardNumberUtils.swift

```diff
@@ -0,0 +1,283 @@
+//
+//  BankCardNumberUtils.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 30/11/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+
+struct BankCardNumberUtils {
+
+    struct IinRangeInfo {
+
+        let range: CountableClosedRange<Int>
+
+        let cardBrand: BankCardBrand
+
+        let maxCardNumberLength: Int
+
+        static func info(_ range: CountableClosedRange<Int>, _ cardBrand: BankCardBrand, _ maxCardNumberLength: Int) -> IinRangeInfo {
+            return self.init(
+                range: range,
+                cardBrand: cardBrand,
+                maxCardNumberLength: maxCardNumberLength
+            )
+        }
+
+    }
+
+    static func cardNumberDigitsString(
```

Comment:
Non-string on input, non-string in output.
Does String.UnicodeScalarView is fully String compatible?